src/components/icons/social-icon.component.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Icon, IconProps } from '@chakra-ui/react';
+import { TikTokIcon } from './social';
```

Comment:
Just to follow the existing pattern shouldn't this be imported from `from '../icons';`?

Reply:
Thank you for noticing it, completely missed this one.